playground/nextjs/src/Header.tsx

```diff
@@ -25,6 +25,7 @@ export const PageHeader = () => {
                         <Link href="/replay-examples/canvas">Canvas</Link>
                         <Link href="/replay-examples/media">Media</Link>
                         <Link href="/replay-examples/long">Long</Link>
+                        <Link href="/EcommerceTester">E-commerce</Link>
```

Comment:
Cursed capitalization of `ecommerce`

Reply:
Ah I think my IDE did this when I renamed the react component, yeah agreed that this is cursed